### PR TITLE
[tflchef] Support CumSum operation

### PR DESCRIPTION
**Pull request by batcheu**
This adds support for CumSum operation in tflchef.

ONE-DCO-1.0-Signed-off-by: Jonghwa Lee <jonghwa3.lee@samsung.com>

---
Realated w/ : https://github.com/Samsung/ONE/issues/11842
Draft PR : https://github.com/Samsung/ONE/pull/11810

### CODE REVIEW

**Comment by hseok-oh** [comment]
@nnfw-bot test all

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/tflchef/proto/tflchef.proto [hunk 1]
```diff
@@ -274,6 +274,11 @@ message CosOptions {
   // None
 }
 
+message CumSumOptions {
+  optional bool exclusive = 1;
+  optional bool reverse = 2;
```
Comment (shs-park) [inline compiler/tflchef/proto/tflchef.proto:279]: The CumSum `reverse` option seems to have a default value - false.
How about adding it? =)

```suggestion
  optional bool exclusive = 1;
  optional bool reverse = 2 [default = false];
```

Note: https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/cumsum

File: compiler/tflchef/tflite/src/Op/include/CumSum.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+{
+public:
+  void filler(const tflite::Operator *op, TFliteImport *import,
+              tflchef::ModelRecipe *model_recipe) const override;
+  tflchef::Operation *build(const tflite::Operator *op, TFliteImport *import,
+                            tflchef::ModelRecipe *model_recipe) const override;
+};
+
+} // namespace tflchef
+#endif // __TFLITE_OP_CUMSUM_H__
```
Comment (shs-park) [inline compiler/tflchef/tflite/src/Op/include/CumSum.h:38]: It's not a big deal, but...
How about adding an empty line for consistency with the other header files? =)

```suggestion
} // namespace tflchef

#endif // __TFLITE_OP_CUMSUM_H__
```

**Review by shs-park — APPROVED** [review]
I left a few comments.
Please take a look.

File: compiler/tflchef/proto/tflchef.proto (hunk 1 above)
Comment (seanshpark) [reply compiler/tflchef/proto/tflchef.proto:279]: I also think adding default looks good :)

Comment (batcheu) [reply compiler/tflchef/proto/tflchef.proto:279]: Thanks, I'll apply it. 👍

File: compiler/tflchef/tflite/src/Op/include/CumSum.h (hunk 2 above)
Comment (batcheu) [reply compiler/tflchef/tflite/src/Op/include/CumSum.h:38]: ditto 👍

File: compiler/tflchef/proto/tflchef.proto (hunk 1 above)
Comment (batcheu) [reply compiler/tflchef/proto/tflchef.proto:279]: FYI, the default value of boolean type in protocol buffer v2 is `false`.
https://protobuf.dev/programming-guides/proto2/#optional:~:text=For%20bools%2C%20the%20default%20value%20is%20false.
Even though, the explicit assignment of default value may be helpful for preventing human errors.

In addition to, Protocol buffer v3 doesn't allow non-zero default value 🤔.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**